packages/core/lib/console.js

```diff
@@ -274,22 +274,21 @@ class Console extends EventEmitter {
       // async function or not. If our script is an async function,
       // this will ensure the console waits until that await is finished.
       Promise.resolve(runScriptResult)
-        .then(value => {
-          // If there's an assignment to run, run that.
-          if (assignment) return runScript(vm.createScript(assignment));
-          return value;
-        })
         .then(value => {
           // All good? Return the value (e.g., eval'd script or assignment)
           runScript(
             vm.createScript(
-              'console.log("Entered expression evaluates to promise; auto-resolving:")'
+              'console.log("The expression evaluates to a Promise; ...")'
```

Comment:
Sorry, I didn't literally mean for the "..." to make it in there.

The text I was trying to suggest was "The expression evaluates to a Promise; auto-resolving:"

Sorry for not being more clear.

Reply:
updated. thanks